plugin/unifyQC.mli.cppo

```diff
@@ -30,11 +30,19 @@ val umfind : UM.key -> 'a UM.t -> 'a
 val lookup : unknown -> umap -> range option
 module OrdTSS :
   sig type t = unknown * unknown val compare : 'a -> 'a -> int end
+#if COQ_VERSION >= (8, 20, 0)
```

Comment:
Breaks QuickChick on Coq 8.20.

Maybe worth mentioning the expected Coq version to appear in future PR descriptions.

Reply:
Indeed the related Coq PR was only for 8.21, sorry. I'll try to think about mentioning the target version in the future.